audioSetup.js

```diff
@@ -0,0 +1,139 @@
+
+// En este documento podríamos agregar entradas y salidas si quisieramos usar el mic de alguna manera 
+
+function AudioSetup(){
+    
+    var self = this;
```

Comment:
Hay que tener cuidado con la palabra "this" en JS, el valor de esta depende de como invoque la funcion, en este caso "this" seria el objeto window lo cual la convierte en una variable global y esto puede causar problemas, mañana podemos revisarlo a mas detalle.
https://dev.to/okrohan/javascript-the-this-keyword-context-cheatsheet-2361